docs/clients/multi_turn_angle_control.rst

```diff
@@ -164,21 +164,21 @@ Type ID 59 | Multi-turn Angle Controller
 | 23     | trajectory_linear_displacement  | get, set       | float     | m                 | Final absolute displacement of trajectory.                                                       |
 +--------+---------------------------------+----------------+-----------+-------------------+--------------------------------------------------------------------------------------------------+
 | 24     | trajectory_linear_velocity      | get, set       | float     | m/s               | Final velocity of the trajectory. Defaults to 0.                                                 |
 +--------+---------------------------------+----------------+-----------+-------------------+--------------------------------------------------------------------------------------------------+
-| 25     | trajectory_linear_acceleration  | get, set       | float     | .. math:: m/s^2   | Final acceleration of the trajectory. Defaults to 0.                                             |
+| 25     | trajectory_linear_acceleration  | get, set       | float     | m/s^2             | Final acceleration of the trajectory. Defaults to 0.                                             |
 +--------+---------------------------------+----------------+-----------+-------------------+--------------------------------------------------------------------------------------------------+
-| 26     | trajectory_average_speed        | get, set       | float     | .. math:: m/s^2   | Average speed of a trajectory. Trajectory is executed or queued once this is sent. Must be $>0$. |
+| 26     | trajectory_average_speed        | get, set       | float     | rad/s             | Average speed of a trajectory. Trajectory is executed or queued once this is sent. Must be >0.   |
 +--------+---------------------------------+----------------+-----------+-------------------+--------------------------------------------------------------------------------------------------+
-| 27     | trajectory_queue_mode           | get, set, save | int8      | .. math:: m/s^2   | append=0, overwrite=1                                                                            |
+| 27     | trajectory_queue_mode           | get, set, save | int8      | enum              | append=0, overwrite=1                                                                            |
 +--------+---------------------------------+----------------+-----------+-------------------+--------------------------------------------------------------------------------------------------+
-| 29     | ff                              | get, set       | uint32    |                   | Feed forward term                                                                                |
+| 29     | ff                              | get, set       | uint32    | V                 | Feed forward term                                                                                |
```

Comment:
This was missing a unit. 
![image](https://github.com/iq-motion-control/iq-module-communication-doc/assets/120586722/95b18fed-96d7-42d6-8168-6e216140dd2b)
ff represents u, the feed forward value in our PID value, aka an added voltage.